front-end/src/app/shared/components/transaction-type-base/transaction-type-base.component.ts

```diff
@@ -195,6 +196,10 @@ export abstract class TransactionTypeBaseComponent implements OnInit, OnDestroy
     });
     payload.contact_id = payload.contact?.id;
 
+    if (payload.children) {
+      payload.children = this.updateChildPurposeDescriptions(payload as SchATransaction);
```

Comment:
@Elaine-Krauss-TCG  Since this functionality is exclusive the the contribution_purpose_description of the Schedule A transaction, the updateChildPurposeDescription method (and renamed "updateChildren()") should be moved to the SchATransaction model and called from there. To call it, we could generalize a new method on the Transaction class called "updateChildren()" that does nothing in the Transaction class but then is overridden in the ScheduleATransaction class with the method below.